pkg/controller/linstorcsidriver/linstorcsidriver_controller.go

```diff
@@ -676,8 +676,21 @@ func newCSINodeDaemonSet(csiResource *piraeusv1.LinstorCSIDriver) *appsv1.Daemon
 		Resources: csiResource.Spec.Resources,
 	}
 
+	linstorWaitNodeInitContainer := corev1.Container{
+		Name:            "linstor-wait-node-online",
+		Image:           csiResource.Spec.LinstorPluginImage,
```

Comment:
Wouldn't it be better to use separate `linstor-wait-until` image for all components instead of putting its binary into the each component image?

Reply:
Another image is another component that needs to be configured, another component that users might need to override/replace. then you have to do this weird song&dance with upgrade CRDs, because that's were they need to be configured, and you might need a default value somewhere in the operator.

so I just skip all that, put it into the containers that need to be downloaded in any case and be done with it.